.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,79 @@
+name: Build Container And Run Integration Tests
+on:
+  schedule:
+    - cron: "0 12 * * 6"  # every Saturday at 12 pm UTC
+  workflow_dispatch:
+    inputs:
+      backend-image:
+        default: "ghcr.io/ust-quantil/qhana-backend:main"
+        required: false
+        type: string
+      ui-image:
+        default: "ghcr.io/ust-quantil/qhana-ui:main"
+        required: false
+        type: string
+      registry-image:
+        default: "ghcr.io/ust-quantil/qhana-plugin-registry:main"
+        required: false
+        type: string
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: ${{ github.repository }}
+
+jobs:
+  image:
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'workflow_dispatch' }}
+    permissions:
+      contents: read
+      packages: write
+    outputs:
+      image: ${{ steps.image_ref.outputs.lowercase }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+      - name: Log in to the Container registry
+        uses: docker/login-action@v2
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build and push Docker image
+        uses: docker/build-push-action@v4
+        id: build_push
+        with:
+          context: .
+          platforms: linux/amd64
+          push: true
+          tags: ${{ steps.meta.outputs.tags }}
```

Comment:
Can you add a tag or label for integration test (so that we can identify and remove them easily)? Also check if these images may pile up to create issues for us later.